.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true;
+
+[*]
+indent_style = space
```

Comment:
Sorry, I based it on the LogViewer. But since that was spaced you also used spaces there. Will change it.